From ae045c7dc8a188e01436535280f40928cb9a26c4 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Wed, 17 Jan 2018 23:34:58 +0000
Subject: [PATCH] Do not build the Printers panel

Continuous does not provide Samba, which is a required dependency of the
Printers panel.
---
 panels/meson.build      | 1 -
 shell/cc-panel-loader.c | 2 --
 2 files changed, 3 deletions(-)

diff --git a/panels/meson.build b/panels/meson.build
index d02714733..24b084405 100644
--- a/panels/meson.build
+++ b/panels/meson.build
@@ -11,7 +11,6 @@ panels = [
   'notifications',
   'online-accounts',
   'power',
-  'printers',
   'privacy',
   'region',
   'search',
diff --git a/shell/cc-panel-loader.c b/shell/cc-panel-loader.c
index 675833c12..b83ee59e1 100644
--- a/shell/cc-panel-loader.c
+++ b/shell/cc-panel-loader.c
@@ -48,7 +48,6 @@ extern GType cc_wifi_panel_get_type (void);
 extern GType cc_notifications_panel_get_type (void);
 extern GType cc_goa_panel_get_type (void);
 extern GType cc_power_panel_get_type (void);
-extern GType cc_printers_panel_get_type (void);
 extern GType cc_privacy_panel_get_type (void);
 extern GType cc_region_panel_get_type (void);
 extern GType cc_search_panel_get_type (void);
@@ -93,7 +92,6 @@ static struct {
   PANEL_TYPE("notifications",    cc_notifications_panel_get_type),
   PANEL_TYPE("online-accounts",  cc_goa_panel_get_type          ),
   PANEL_TYPE("power",            cc_power_panel_get_type        ),
-  PANEL_TYPE("printers",         cc_printers_panel_get_type     ),
   PANEL_TYPE("privacy",          cc_privacy_panel_get_type      ),
   PANEL_TYPE("region",           cc_region_panel_get_type       ),
   PANEL_TYPE("search",           cc_search_panel_get_type       ),
-- 
2.14.3

